functional_tests/metaflow/wandb-example-flow-decoboth.py

```diff
@@ -1,24 +1,5 @@
 """
 Test Metaflow Flow integration
-
----
-id: metaflow.example.both
-plugin:
-  - wandb
-command:
-    args:
-        - --no-pylint
-        - run
-assert:
-    - :wandb:runs_len: 4
-    - :wandb:runs[0][config]: {"seed": 1337, "test_size": 0.2, "raw_data": https://gist.githubusercontent.com/tijptjik/9408623/raw/b237fa5848349a14a14e5d4107dc7897c21951f5/wine.csv}
-    - :wandb:runs[1][config]: {"seed": 1337, "test_size": 0.2, "raw_data": https://gist.githubusercontent.com/tijptjik/9408623/raw/b237fa5848349a14a14e5d4107dc7897c21951f5/wine.csv}
-    - :wandb:runs[2][config]: {"seed": 1337, "test_size": 0.2, "raw_data": https://gist.githubusercontent.com/tijptjik/9408623/raw/b237fa5848349a14a14e5d4107dc7897c21951f5/wine.csv}
-    - :wandb:runs[3][config]: {"seed": 1337, "test_size": 0.2, "raw_data": https://gist.githubusercontent.com/tijptjik/9408623/raw/b237fa5848349a14a14e5d4107dc7897c21951f5/wine.csv}
-    - :wandb:runs[0][exitcode]: 0
-    - :wandb:runs[1][exitcode]: 0
-    - :wandb:runs[2][exitcode]: 0
-    - :wandb:runs[3][exitcode]: 0
```

Comment:
shy move it to a separate file?

Reply:
because the py file could serve as cleaner documentation without all the yea markup.